src/v2/System/Router/RenderStatus.tsx

```diff
@@ -50,15 +49,11 @@ export const RenderPending = () => {
   }
 }
 
-// @ts-expect-error STRICT_NULL_CHECK
-export const RenderReady: React.FC<{
-  elements: React.ReactNode
-}> = props => {
+export const RenderReady = (props: { elements: React.ReactNode }) => {
```

Comment:
There's an unfortunate property about `found` where in order to trigger certain router behaviors one has to return `undefined` (vs `null`). If we're currently fetching we return `undefined` here, which keeps the render pending component running smoothly and displaying the top loading indicator bar.